README.md

```diff
@@ -198,6 +197,20 @@ Installation
 
 ### macOS
 
+#### Quick start, installing all completers
+
+- Install cmake, macvim and python
+
+    brew install cmake macvim python
```

Comment:
I think worth clarifying that in addition to MacVim, Vim and NeoVim from Homebrew are suitable. If I'm not mistaken, Vim previously failed for YouCompleteMe because it was built without Python 2 support. But for now, in YouCompleteMe support Python 2 has been dropped.